docs/source/contents/4_draft_mapping.rst

```diff
@@ -2620,9 +2620,9 @@ table of the values that need to be edited along with their appropriate match.
 
 .. code-block:: turtle
 
-    @prefix schema: <http://schema.org/> .
+    @prefix dcterms: <http://purl.org/dc/terms/> .
```

Comment:
Before this, (in justification or decision), explain the decision that we are ignoring the `http://purl.org/dc/dcam/rangeIncludes` since it is only a suggestion.  Don't care why, but I'd explain since it's a little grey area for the philosophy.

Reply:
Thanks. I attempted this through the text "we can disregard the recommendation in `dcterms:temporal` to enter values appropriate to the class PeriodOfTime (that have both start and end dates)." I'll be more explicit.